src/pages/Buy/Buy.tsx

```diff
@@ -98,20 +105,27 @@ export const Buy = () => {
               alignItems={alignItemsXlFlexStart}
               textAlign={textAlignXlLeft}
             >
-              <Heading
+              <Text
+                as='h2'
                 fontSize={headingFontSize}
                 lineHeight='1em'
                 letterSpacing='-0.05em'
                 color='whiteAlpha.900'
-              >
-                {translate('buyPage.title.first')}{' '}
-                <Text
-                  as='span'
-                  background='linear-gradient(97.53deg, #F687B3 5.6%, #7B61FF 59.16%, #16D1A1 119.34%)'
-                  backgroundClip='text'
-                  translation='buyPage.title.second'
-                />
-              </Heading>
+                translation='buyPage.title'
+                components={useMemo(
+                  () => ({
+                    span: (
+                      <RawText
+                        as='span'
+                        background='linear-gradient(97.53deg, #F687B3 5.6%, #7B61FF 59.16%, #16D1A1 119.34%)'
+                        backgroundClip='text'
+                      />
+                    ),
+                  }),
+                  [],
+                )}
```

Comment:
Suggestion: this should probably be extracted.

Reply:
Indeed, done!